app/views/channels/_text_components.html.haml

```diff
@@ -0,0 +1,22 @@
+- @channel.text_components.each do |text_component|
```

Comment:
Because you need a view where you can see all text_components in a channel. This is pretty much what the cucumber feature asks for.